packages/opentelemetry-instrumentation-watsonx/tests/cassettes/test_generate/test_generate.yaml

```diff
@@ -0,0 +1,615 @@
+interactions:
+- request:
+    body: apikey=test_api_key&grant_type=urn%3Aibm%3Aparams%3Aoauth%3Agrant-type%3Aapikey
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '111'
+      Content-Type:
+      - application/x-www-form-urlencoded
+      User-Agent:
+      - python-requests/2.31.0
+    method: POST
+    uri: https://iam.cloud.ibm.com/oidc/token
+  response:
+    body:
+      string: '{"access_token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoibm9uZSIsInN1YiI6Im5vb25lQGlibS5jb20iLCJpYW1faWQiOiJJQk1pZC0xMDAwMDBQVzAwIiwiYWNjb3VudCI6eyJic3MiOiJhYmMxMjMifSwiaWF0IjoxNzA4NTkzNzM3LCJleHAiOjIwMjM5NTM3Mzd9.iYXuVHrO3J-InoMRvwM2ENUlUWsiLzut_9wo97McECU","refresh_token":"eyJhbGciOiJydCJ9.eyJpYW1faWQiOiJJQk1pZC0xMTAwMDBTRFMxIiwiYWNjb3VudF9pZCI6ImQ3NTY2MTc1NzVmZjYzMWYzMmFiYzkzNzI1ZDIxYjdjIn0.f8p9rFmcUEa1cBeBteWI1upnP5l4f9dysuKKFn6tO83FLZBDvIi3UubtxVdgmGh7VfB2ZgmgpIE0ZyPYVmJef3JfAGEJYkvC87xPzrCVyBSlnoRrQUW6oBfJVQYZEsFtd9lVJlN0-HCCLNR0aFnEPKBNfBM1YA2enkq5aSL6NjTp_JJR4SadpchKNmeUj9HIzoAGU8sRSHr5rHS156hUwc5W-ecZflS_KFLEmE-Zv1zNZ6jTuRF4bLmLKIPK-9pODu7wFEMi5GX3KbTMbTs1Vgg9i6bPso_8FWS2Wv29cjMrIEe2fMZBAGP4d6BYicITrExJoOblWwRyIq-ahPSYWwJ-dJI6lbYXefBtK2Om-7a8NtuQ8a1qRlL5NHmD8Ks42GxuRzRiGv8lV_OdySgBSm6cLLgPquLTU8ERreKnZeV6b9e8P-UiYEaWgz1HvvWAproZFu9e0zCWwPQU9dR2ZpecEssX9t0O9Eo1lnittvv_LpXR5yOISMKuYfT8ipawyOxNtGE7fuh05sBrzsmj0Uw1auy36uojzMKO4oSqkDI6mTFn_z0VBOIY1Zh3ly0g9VQEQlFwt-sWK2MshFD2Hjez7-AWT_ydorFL2497TGBn5JpiNcdExlZYXuRGXl4HY5c-sWbdVNb14Yhft1h6Fof0YhASnwH0k3JCQfo_ec_P6NMZeypK-fZLe5ma4H3Z1lo3Jji7J0zMwbPpWjcUheUlgLqJb2Pyr3k9GQxu9bErnHShIUdxM-4x3JNnKe-PwwyxMW9VR_cglwnYfWqflXBtttnmYGgSKc5KEZMuY6-Jj6HhwDF4RUlc6mFK_v_1KVtQUyULZbGX6ctZKBxSysgl6p7mbDbzdLwLNZisulITyCt-LQzWsLL4WWOQxRQF-UykGR63_6temZ--EJi9wRmm9fJ5qdfCHdsI9v1pA3LOuNFsPfJE63TeYCdG2kf7JF_2ElLo_IaQTDQ5TunU4GV-LLoc6b-7dhFdI8fQfGvO4ZPWLFQZ9lCfDUiGmbkEWUhgy59lWtKVuqpVB2qJWKYwUg-P41m_zFlIETj4sz0ELNtR32H6F9I5A6F5vvX2P30wAvW2PBsSnWE474fb5EUI_ejweOyEz-067Qis4JIUsguIbmI8-JDuMDobSbyFA2_ujkUBjcbRqrQs4bY86hbu-w_-Wqf1qbKYxpV8Mb3SQLM0PaCAhCjbq9o5gcfhUhP-9HWgCla04BVTPqrf7nJc270bcyKb_HDar3SiTYgxDe-WFCXP2uX_QS1EziqkP-uPvnlaz1yjC8Uxw76MzuTbmPxllcH8RuOnAz2ai-9nXKM2RiN06D62bE0mc-NB1k1Q8ZYr6YsW-rAPIJ6BC8NL","token_type":"Bearer","expires_in":3600,"expiration":1708597208,"refresh_token_expiration":1708852808,"scope":"ibm
```

Comment:
❓ Are you sure it’s OK (from your side) committing these credentials here?

I’m a bit short on time to check carefully, maybe you’re missing a VCR filter for this? (Hence the failure in CI/CD)?

Reply:
@paolorechia this is actually a test jwt token I generated, not the actual one, and there're other api keys in this yaml, but there were long expired (within an hour), this yaml was recorded some hours ago, so no need to worry about.

Reply:
those failures were caused by a token self validation, b/c the jwt token encoded with an expiration time(an hour), so I workaround it by creating a test token that never expire.

Reply:
Nice find! I saw now they passed